lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSBottomSheetService {
+  final BuildContext context;
+  final Widget child;
+
+  DSBottomSheetService({
+    required this.context,
+    required this.child,
+  });
+
+  Widget _buildBottomSheet({ScrollController? controller}) {
+    return Container(
+      decoration: const BoxDecoration(
+        color: DSColors.neutralLightSnow,
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(15.0),
+          topRight: Radius.circular(15.0),
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: 16.0,
+          vertical: 8.0,
+        ),
+        child: Stack(
```

Comment:
I feel like there should have more bottom padding between the grabber and the content.

![image](https://user-images.githubusercontent.com/91232521/195175815-eae4c149-c4b9-4159-bebb-0737300c8b5b.png)

![image](https://user-images.githubusercontent.com/91232521/195175875-d47ccc3f-0e5e-4012-81dd-81cdcf7ca9f8.png)

![image](https://user-images.githubusercontent.com/91232521/195175947-5ae99b02-35ad-46f8-8e89-5cd7bb1d7975.png)

Reply:
done